js/src/lib/components/InferenceWidget/widgets/AudioClassificationWidget/AudioClassificationWidget.svelte

```diff
@@ -163,6 +164,7 @@
 	{computeTime}
 	{error}
 	{isLoading}
+	bind:isDisabled
```

Comment:
You should use `let:isDisabled` I think

https://svelte.dev/docs/special-elements#slot-slot-key-value

Reply:
great suggestion. Makes the code cleaner c6bd915804c0459aca9e6916e58cadd59b771f37